tools/config.py

```diff
@@ -3,15 +3,15 @@
 
 # Update with the Meilisearch version TAG you want to build the AMI with
 
-MEILI_CLOUD_SCRIPTS_VERSION_TAG = 'v0.25.0'
+MEILI_CLOUD_SCRIPTS_VERSION_TAG = 'v0.25.2'
 
 # Update with the AMI id that you want to publish after TESTING
 
-PUBLISH_IMAGE_ID = 'ami-06bc537d7c0c3c53c'
+PUBLISH_IMAGE_ID = 'ami-0196ad076af7e4f20'
 
 # Update with the AMI name that you want to unpublish/delete worldwide
 
-DELETE_IMAGE_NAME = 'MeiliSearch-v0.23.1-Debian-10.3'
+DELETE_IMAGE_NAME = 'MeiliSearch-v0.22.0-Debian-10.3'
```

Comment:
should it not be 24 ?

Reply:
Last time I delete v0.23.1 instead of v0.22.0 so I do it this time.
There remains the:
v0.24.0
v0.25.0
v0.25.2
Because I prefer to let the last two versions in addition to the actual version.
![Screenshot 2022-01-31 at 13 12 56](https://user-images.githubusercontent.com/16155041/151792241-64e90472-da6d-4404-b452-42e866f65612.png)

Reply:
Okey! Not sure I understand but I trust you

Reply:
@bidoubiwa,
This is the number of the version you want to delete. When I publish a new version I normally delete the oldest version published trying to always leave the users the last 2 versions available.
I should delete the v0.22.0 when I cleaned last time but instead, I delete the v0.23.2.
Next release I will delete v0.24.0, like that the users could have access to the v0.25.0, v0.25.2, and v0.26.0. In case he needs an older version due to Breaking Changes or specific bugs.